src/Iterator/GlobIterator.php

```diff
@@ -33,9 +33,12 @@ class GlobIterator extends IteratorIterator
      *
      * @param string $glob  The glob pattern.
      * @param int    $flags A bitwise combination of the flag constants in
-     *                      {@link Glob}.
+     *               {@link Glob}.
+     * @param bool   $skipErrors Add the RecursiveIteratorIterator::CATCH_GET_CHILD
+     *               to internal RecursiveIteratorIterator and prevent throwing
+     *               Exception on errors like access denied.
      */
-    public function __construct($glob, $flags = 0)
+    public function __construct(string $glob, int $flags = 0, bool $skipErrors = false)
```

Comment:
Hmm, we can't really enforce `string $glob` and `int $flags` here, as it would break BC for downstream consumers :thinking: 

The type additions can only really be safely added to the new constructor argument that you added